etl/storage/storage.go

```diff
@@ -103,6 +105,18 @@ func filterByPulse(query *gorm.DB, pulseNumberLt, pulseNumberGt *int) *gorm.DB {
 	return query
 }
 
+func filterByJetDropID(query *gorm.DB, jetDropIDGte, jetDropIDLte *models.JetDropID) *gorm.DB {
+	if jetDropIDGte != nil {
+		query = query.Where("(pulse_number >= ? and jet_id >= ?)", jetDropIDGte.PulseNumber, jetDropIDGte.JetID)
```

Comment:
if `jetDropIDGte`="0001:12345", we don't get jet drop with id="0000:12346" with this query